index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Oswald&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/4fd73de0d8.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Calisthenics Home</title>
+</head>
+<body>
+    <header>
+        <nav id="top-header" class="navbar navbar-expand-md d-none d-md-flex">
+            <div class="collapse navbar-collapse">
+                <ul class="navbar-nav ms-md-auto">
+                    <li class="nav-item me-1">
+                        <a href="#" class="nav-link">
+                            <i class="fab fa-facebook-f text-white "></i>
+                        </a>
+                    </li>
+                    <li class="nav-item me-4">
+                        <a href="#" class="nav-link">
+                            <i class="fab fa-twitter text-white"></i>
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link text-white">
+                           Logout 
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link text-white">
+                            Mypage
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link text-white pe-md-2">
+                            Spanish
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <nav id="bottom-header" class="navbar navbar-expand-md">
```

Comment:
[Optional]
On the design template, the top dark section of the menu is shorter than the bottom white section. It will be better if you reduce the height of the top menu a little bit. (In order to make it similar to the design template)